Doc/howto/clinic.rst

```diff
@@ -392,9 +394,10 @@ Let's dive in!
         Write a pickled representation of obj to the open file.
         [clinic start generated code]*/
 
-12. Save and close the file, then run ``Tools/clinic/clinic.py`` on
-    it.  With luck everything worked---your block now has output, and
-    a ``.c.h`` file has been generated! Reopen the file in your
+12. Save and close the file, then run ``make clinic``
```

Comment:
Side note: running the script on the specified file and running `make clinic` is not the same. The latter may be more correct, but it is slower.

Reply:
I know, but generally it's better if contributors use `make clinic`, so I think it makes sense to use that throughout the tutorial. I think the "basic usage" section could be improved to mention how to run clinic directly on a specified file.

Reply:
I decided to revert these changes as well; we can change it to the Make target in a future PR.